apps/src/lib/kits/maker/ui/SetupGuide.jsx

```diff
@@ -75,13 +114,31 @@ export default class SetupGuide extends React.Component {
       );
     }
 
+    // Experiment 'microbit' renders additional description.
+    let isMicrobit = experiments.isEnabled('microbit');
+
     if (isCodeOrgBrowser() || isChromeOS() || isWebSerial) {
       return <SetupChecklist webSerialPort={webSerialPort} />;
     }
     return (
-      <Provider store={store}>
-        <Downloads />
-      </Provider>
+      <div>
+        <h1>{applabI18n.makerSetupPageTitle()}</h1>
+        <div style={isMicrobit ? style.boardDescriptions : {}}>
+          <Description
+            {...setupGuideContent.circuitPlayground}
+            divStyle={isMicrobit ? style.descriptionFlexCard : {}}
+          />
+          {isMicrobit && (
```

Comment:
Not a blocking change. I don't mind it. A curiosity. Does this change (and use of `isMicrobit` here) indicate that there might be more than one and perhaps more than two at some point? Is this the chance to properly generalize any number of descriptions instead of having two+ render paths?